code/modules/client/preference_setup/loadout/loadout_accessories.dm

```diff
@@ -167,64 +167,16 @@
 	path = /obj/item/clothing/accessory/horrible
 
 /datum/gear/accessory/scarf
-	display_name = "scarf"
+	display_name = "scarf selection"
 	path = /obj/item/clothing/accessory/scarf
 
-/datum/gear/accessory/scarf/red
-	display_name = "scarf, red"
-	path = /obj/item/clothing/accessory/scarf/red
-
-/datum/gear/accessory/scarf/green
-	display_name = "scarf, green"
-	path = /obj/item/clothing/accessory/scarf/green
-
-/datum/gear/accessory/scarf/darkblue
-	display_name = "scarf, dark blue"
-	path = /obj/item/clothing/accessory/scarf/darkblue
-
-/datum/gear/accessory/scarf/purple
-	display_name = "scarf, purple"
-	path = /obj/item/clothing/accessory/scarf/purple
-
-/datum/gear/accessory/scarf/yellow
-	display_name = "scarf, yellow"
-	path = /obj/item/clothing/accessory/scarf/yellow
-
-/datum/gear/accessory/scarf/orange
-	display_name = "scarf, orange"
-	path = /obj/item/clothing/accessory/scarf/orange
-
-/datum/gear/accessory/scarf/lightblue
-	display_name = "scarf, light blue"
-	path = /obj/item/clothing/accessory/scarf/lightblue
-
-/datum/gear/accessory/scarf/white
-	display_name = "scarf, white"
-	path = /obj/item/clothing/accessory/scarf/white
-
-/datum/gear/accessory/scarf/black
-	display_name = "scarf, black"
-	path = /obj/item/clothing/accessory/scarf/black
-
-/datum/gear/accessory/scarf/zebra
-	display_name = "scarf, zebra"
-	path = /obj/item/clothing/accessory/scarf/zebra
-
-/datum/gear/accessory/scarf/christmas
-	display_name = "scarf, christmas"
-	path = /obj/item/clothing/accessory/scarf/christmas
-
-/datum/gear/accessory/scarf/stripedred
-	display_name = "scarf, striped red"
-	path = /obj/item/clothing/accessory/stripedredscarf
-
-/datum/gear/accessory/scarf/stripedgreen
-	display_name = "scarf, striped green"
-	path = /obj/item/clothing/accessory/stripedgreenscarf
-
-/datum/gear/accessory/scarf/stripedblue
-	display_name = "scarf, striped blue"
-	path = /obj/item/clothing/accessory/stripedbluescarf
+/datum/gear/accessory/scarf/New()
+	..()
+	var/list/scarves = list()
+	for(var/scarf in typesof(/obj/item/clothing/accessory/scarf/))
```

Comment:
If you do this, you need to give the base scarf item a sprite; this call includes the base item in it's list, and it doesn't look like /obj/item/clothing/accessory/scarf overwrites the tie sprite that is the base for accessories.

Reply:
...huh. I just tested that the selection worked and let you pick different ties- I tested 3 of them, but not the base one. My bad. I'll adjust that.

Reply:
Yeah, easy to overlook.